eventsourcing/src/test/java/org/axonframework/eventsourcing/DefaultSnapshotterSpanFactoryTest.java

```diff
@@ -0,0 +1,83 @@
+package org.axonframework.eventsourcing;
+
+import org.axonframework.tracing.IntermediateSpanFactoryTest;
+import org.axonframework.tracing.SpanFactory;
+import org.axonframework.tracing.TestSpanFactory;
+import org.junit.jupiter.api.Test;
+
+
+class DefaultSnapshotterSpanFactoryTest extends IntermediateSpanFactoryTest<DefaultSnapshotterSpanFactory.Builder, DefaultSnapshotterSpanFactory> {
+
+    @Test
+    void testCreateScheduleSnapshotSpanWithDefaults() {
```

Comment:
I believe that with the change from JUnit 4 to JUnit 5, we complied with the format that test methods are not prefixed with "test".